src/util/unicode.cpp

```diff
@@ -298,6 +301,28 @@ static void utf16_native_endian_to_java(
   }
 }
 
+/// Additionally escapes the '\'' character for java characters
```

Comment:
Say what it does / what it's for, probably something like "escapes non-printable characters, whitespace except for spaces, double- and single-quotes and backslashes. This should yield a valid Java identifier"